client/src/components/modals/incorporate-login.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <v-dialog v-model="showModal" max-width="60%">
```

Comment:
You might want to attach this to the parent page for ease of testing.
You might want to make this persistent (ie, can't close when clicking off of it)?

Reply:
All of the dialogs are mounted in the `App.vue` and to change the pattern would require refactoring all of them. The same goes for the persistent prop. 
In this specific case, the common component we're using within this modal doesn't have a close action and it looked a little out of place when layered on top of the common component.

Reply:
They're declared in App.vue but they are mounted near the top of the DOM, so testing them in the context of a page/view can be difficult since you can't mount them (Vue test utils only mounts the page/view component, not the entire app).
FWIW and apparently not a problem atm.

Reply:
Absolutely agree, to all the above.